database/migrations/2022_11_21_200442_create_destinatarios.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('destinatarios', function (Blueprint $table) {
+            $table->integer ('id_destinatario')->primary();
+            $table->integer('id_cliente');
+            $table->text('num_pedido');
```

Comment:
No se pone integer para poner llave foránea de cliente y de pedidos
https://laravel.com/docs/9.x/migrations#column-method-foreignId

No usen text usen string porque text es para textos largos y no tiene caso en el correo electronico 
https://laravel.com/docs/9.x/migrations#column-method-string